spec/factories/users.rb

```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+  sequence :email do |n|
+    "person#{n}@example.com"
+  end
+end
+
+FactoryGirl.define do
+  factory :user, class: "User" do
+    email
+    password { 'asdfjkl;' }
+    password_confirmation { 'asdfjkl;' }
```

Comment:
This might be better as

```
password { 'asdfjkl;' }
password_confirmation { password }
```

So that we can create a valid user with just `create(:user, password: 'test123')`